### ISSUE TITLE
Hung on updating

### ISSUE THREAD

**Issue by sharonmarks** [envato/wp-envato-market#44]
The plugin installed properly and I got my Global OAuth Personal Token but for some reason the update won't finish. It says updating but it has been running for over an hour. That cannot be right.  It is the Montana wordpress theme that I am trying to update.  Thanks!

**Comment by capuderg** [comment]
Hi,

we also experienced this issue. We had a child theme active and wanted to update the parent theme. When we clicked on the "Update theme" button in the Envato Market page, the spinner kept spinning even though the AJAX call was completed very fast and the theme did not update.

Then I refreshed the page and tried again and this time, the theme actually updated ok.

Some other users reported the same issue...

Take care!

**Comment by valendesigns** [comment]
Reload the page and try again, for some unknown reason sometimes the first attempt to install/update a theme or plugin hangs. I'm not sure of the reason, and is extremely difficult to duplicate and test for.

**Comment by Typiary** [comment]
I had the same issue, particularly from the Themes tab within the the Envato Market page – was taking twenty minutes of "Updating..." before I left the page for the WP Dashboard. There I noticed the "Updates" icon, and when I went to the WP Updates page, I could update the theme within seconds as it should be. It also updated the Themes tab on the Envato Market page to the correct newly-updated theme version. I would guess that, like above, it is an issue of "the first time takes forever, so reload; the second time you update, it works fine.."
